tests/builds/experiments_ymls/build_examples/test_build.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+
+int main(int argc, char **argv){
```

Comment:
I understand this is just a test file but it wouldn't hurt to use some kind of formatting option for this. Also, a c++ main function should always return an `int` as stated in it's declaration therefore I would recommend changing the code to the following (including the removal of header includes not used within this file and flushing the output stream):
```c++
#include <iostream>

int main(int argc, char **argv) {
    std::cout << "Cpp file has been build successfully." << std::endl;
    return 0;
}
```